content/_code-samples/non-fungible-token/py/authorize-minter.py

```diff
@@ -0,0 +1,87 @@
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.models.transactions.nftoken_mint import NFTokenMint, NFTokenMintFlag
+from xrpl.models.transactions.account_set import AccountSet, AccountSetFlag
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.models.requests import AccountNFTs
+from xrpl.clients import JsonRpcClient
+from xrpl.wallet import Wallet
+
+# Assign an authorized NFT minter on your account via AccountSet transaction with flag ASF_AUTHORIZED_NFTOKEN_MINTER
+# Mint a NFT using the minter account on behalf of the issuer account
+# https://xrpl.org/authorize-minter.html#set-minter
+
+seed = ""
+custom_wallet = None
+
+if seed:
+    custom_wallet = Wallet(seed=seed, sequence=0)
+
+# Connect to a testnet node
+print("Connecting to Testnet...")
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Initialize wallet from seed
+issuer_wallet = generate_faucet_wallet(client=client, wallet=custom_wallet)
+issuerAddr = issuer_wallet.classic_address
+
+# Get minter account credentials from the testnet faucet
+print("Requesting address from the Testnet faucet...")
+nftoken_minter_wallet = generate_faucet_wallet(client=client)
+minterAddr = nftoken_minter_wallet.classic_address
+
+print(f"\nMinter Account: {issuerAddr}")
+print(f"          Seed: {issuer_wallet.seed}")
+
+print(f"\nAuthorized Minter Account: {minterAddr}")
+print(f"                     Seed: {nftoken_minter_wallet.seed}")
+
+# Construct AccountSet transaction to authorize another account (Minter Account) to issue NFTs on our behalf
+print(f"\nAuthorizing account {minterAddr} as a NFT minter on account {issuerAddr}...")
+authorize_minter_tx = AccountSet(
+    account=issuerAddr,
+    set_flag=AccountSetFlag.ASF_AUTHORIZED_NFTOKEN_MINTER,
+    nftoken_minter=minterAddr
+)
+
+# Sign authorize_minter_tx using issuer account
+authorize_minter_tx_signed = safe_sign_and_autofill_transaction(transaction=authorize_minter_tx, wallet=issuer_wallet, client=client)
+authorize_minter_tx_signed = send_reliable_submission(transaction=authorize_minter_tx_signed, client=client)
+authorize_minter_tx_result = authorize_minter_tx_signed.result
+print(f"\nAuthorize minter tx result: {authorize_minter_tx_result}")
+
+if authorize_minter_tx_result['meta']['TransactionResult'] == "tesSUCCESS":
+    print(f"\nTransaction was successfully validated, minter {minterAddr} is now authorized to issue NFTs on behalf of {issuerAddr}")
+else:
+    print(f"Transaction failed, error code: {authorize_minter_tx_result['meta']['TransactionResult']}"
+          f"\nMinter {minterAddr} is not authorized to issue NFTS on behalf of {issuerAddr}")
+
+# Construct NFTokenMint transaction to mint a brand new NFT
+print(f"Minting a NFT from the newly authorized account to prove that it works...")
+mint_tx_1 = NFTokenMint(
```

Comment:
I think just `mint_tx` or `mint_nftoken_tx` would be fine since there's only one tx here